src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -169,7 +167,15 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
         <div className="ScheduleItemNG__name">{event.name}</div>
         <div className="ScheduleItemNG__place">
           <span className="ScheduleItemNG__place--location">in</span>{" "}
-          {event.room || eventVenue?.name}
+          {eventVenue?.name}
+          {eventRoom && (
+            <>
+              {", "}
+              <button className="button--a" onClick={enterRoom}>
+                {event.room}
+              </button>
+            </>
```

Comment:
ButtonNG?

Not a requirement, but if it doesn't work out, that knowledge could be useful to update it in the future.

Reply:
i made this button look like `<a>` tag but with onClick property (transformed to `span`)
fixed: https://github.com/sparkletown/sparkle/pull/2183/commits/2be26c4e66c5595df4c893f818cc364b297a21b3

![image](https://user-images.githubusercontent.com/63313732/131529995-7554620e-b1a1-4304-8c65-1588c9aa6921.png)